### DISCUSSION TITLE
What's insecure about remote bases?

### DISCUSSION THREAD

**Discussion by lennartack** [fluxcd/flux2#4824] [Q&A]
[Flux advises to disallow Kustomize remote bases for security reasons.](https://fluxcd.io/flux/faq/#should-i-be-using-kustomize-remote-bases) What are the security risks of remote bases?

I'm investigating a solution for deploying templates to multiple tenants with different Git repositories. Remote bases would avoid the need to copy the templates to every repository.

**Comment by makkes** [comment]
Our [security best practices guide](https://fluxcd.io/flux/security/best-practices/#start-up-flags-1) provides a basic rationale behind this recommendation.

**Reply by stefanprodan** [reply]
> Remote bases would avoid the need to copy the templates to every repository.

No need to duplicate the files, use Flux include feature: https://fluxcd.io/flux/components/source/gitrepositories/#include
